axiom/nr_txn.h

```diff
@@ -389,7 +393,8 @@ void nr_txn_enforce_security_settings(nrtxnopt_t* opts,
  */
 extern nrtxn_t* nr_txn_begin(nrapp_t* app,
                              const nrtxnopt_t* opts,
-                             const nr_attribute_config_t* attribute_config);
+                             const nr_attribute_config_t* attribute_config,
+                             const nrobj_t* log_forwarding_labels);
```

Comment:
These two arguments are not documented by doc string. I don't know if this is something we care about or not.

Reply:
Good point - I addressed this omission with https://github.com/newrelic/newrelic-php-agent/pull/1027/commits/7a02270164facfc55fcdff042cd96e8d6bb5e164